packages/pluggableWidgets/area-chart-web/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "area-chart-web",
+  "widgetName": "AreaChart",
+  "version": "1.0.0",
+  "description": "An area chart displays a solid color between the traces of a graph.",
+  "copyright": "© Mendix Technology BV 2021. All rights reserved.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "config": {
+    "mendixHost": "http://localhost:8080",
+    "developmentPort": 3000
+  },
+  "marketplace": {
+    "minimumMXVersion": "9.6.0",
```

Comment:
@iobuhov @Keraito should we change this property to the latest version? same applies for my #1180

Reply:
9.6.0 should be fine